analyzer/protocol/ipsec/dpd.sig

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
+
+signature ipsec_packet_udp {
+  ip-proto == udp
+
+  # Taken from https://github.com/ukncsc/zeek-plugin-ikev2/blob/master/scripts/dpd.sig
```

Comment:
Same note here as above.

Reply:
I also improved this, so it's not just a copy.

Reply:
I changed the wording around it to sound like I didn't copy verbatim.